lang/en/docs/tutorials/dft/electronic/valence-band-offset.md

```diff
@@ -0,0 +1,101 @@
+The second term, $\Delta V$, is determined from the lineup of the macroscopically averaged electrostatic potential in
+the interface heterostructure.
+
+
+## Choose Materials
+
+When creating the job, the user needs to select **three materials** corresponding to the MoS<sub>2</sub>/WS<sub>2</sub> interface
+and the isolated monolayers of both MoS<sub>2</sub> and WS<sub>2</sub>. Each of the structures is expected to be relaxed.
+
+!!!note "Order of Materials"
+    The VBO workflow assumes the interface structure to correspond to the first material, i.e. please be sure to load the interface
+    structure first.
+
+## Choose Workflow
+
+The [workflow](../../../workflows/overview.md) for calculating the valence band offset can be
+[imported](../../../workflows/actions/copy-bank.md) from the [Workflows Bank](../../../workflows/bank.md) into the account-owned
+[collection](../../../accounts/collections.md).
+This workflow can later be [selected](../../../jobs-designer/actions-header-menu/select-workflow.md) and added to the
+[Job being created](../../../jobs-designer/workflow-tab.md).
+
+The workflow contains two subworkflows per material calculating the valence band maximum (via band structure),
+macroscopically averaged electrostatic potential, and its minima.
+As the system in this tutorial is a heterostructure built of monolayers, determining the value of the macroscopically
+averaged electrostatic potential in the region of the monolayer corresponds to finding the minima of $\overline{V}$.
+For multilayered heterostructures the problem becomes equivalent of finding plateaus of $\overline{V}$.
+
+The final subworkflow collects all the intermediate results and determines the value of the valence band offset.
+
+### Workflow Settings
+
+For the purpose of this tutorial, we set the size of the grid of k-points to 6 x 6 x 1 for each of the three PW-SCF units
+and adjust the k-path to reflect the reduced dimensionality (Γ-M-K-Γ). In addition, one should also adjust the size of the window
+for macroscopic averaging. For the present system we set this size to the distance between the sulfur atoms in both
+monolayers (ca. 5.7 bohr).
+
+## Submit Job
+
+Before [submitting](../../../jobs/actions/run.md) the [job](../../../jobs/overview.md), the user should click on the
+["Compute" tab](../../../jobs-designer/compute-tab.md) of [Job Designer](../../../jobs-designer/overview.md) and inspect
+the [compute parameters](../../../infrastructure/compute/parameters.md) included therein.
+
+## Examine results
+
+When all [unit](../../../workflows/components/units.md) computations are complete at the end of Job execution, switching
+to the [Results tab](../../../jobs/ui/results-tab.md) of [Job Viewer](../../../jobs/ui/viewer.md) will show the results
+of the simulation, including the valence band offset as well as the plots of the planar and macroscopic average of
+the electrostatic potentials.
```

Comment:
We should add picture(s) of the results here.

Reply:
I added the macroscopically averaged potential profiles to the explanation of the potential lineup method